test_scripts/AppServices/GetAppServiceRecords/002_GetAppServiceRecords_HMI.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+--  Precondition: 
+--
+--  Steps:
+--  1) HMI sends a PublishAppService (with {serviceType="MEDIA", handledRPC=ButtonPress})
+--  2) HMI sends a GetAppServiceRecords to Core
+--
+--  Expected:
+--  1) Core returns the service record of the MEDIA service published by the HMI ({servicePublished=true, serviceActive=true}) 
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/AppServices/commonAppServices')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local functions ]]
+local function getAppServiceRecords(serviceType, expectedResponse)
```

Comment:
No need to pass expectedResponse as a parameter when you just call the local function anyways